### PR TITLE
course inputs added

### PR DESCRIPTION
**Pull request by alreuben**
addCourse.js file / addCourse.phtml

Add input boxes

### CODE REVIEW

**Review by Steph-ski — APPROVED** [review]
Looks good, didn't know whether slightly more semantic wording could be used?

File: templates/addCourse.phtml [hunk 1]
```diff
@@ -39,13 +39,25 @@ $trainersCheckbox = \Portal\ViewHelpers\TrainerCheckboxViewHelper::displayTraine
         <label for="course-notes">Notes</label>
         <textarea name="notes" class="create-courses form-control" placeholder="Notes" data-max="5000" maxlength="5000"></textarea>
         <div id="courseNotesError" class="alert hidden formItem_alert"></div>
+
+
```
Comment (Joperes-ym) [inline templates/addCourse.phtml:43]: Clear 42 or 43 empty lines

File: templates/addCourse.phtml [hunk 2]
```diff
@@ -39,13 +39,25 @@ $trainersCheckbox = \Portal\ViewHelpers\TrainerCheckboxViewHelper::displayTraine
...
             <input type="checkbox" id="in_person" value="1" class="create-courses" name="in_person">
+
+            <input id="in_person_input" name="in_person_input" class="create-courses form-control" type="text" data-required="true" />
+            <div id="in_person_input_error" class="alert formItem_alert"></div>
+
         </div>
         <div class="">
             <label for="remote">Remote?</label>
             <input type="checkbox" id="remote" value="1" class="create-courses" name="remote">
+
```
Comment (Joperes-ym) [inline templates/addCourse.phtml:55]: Clear 55 or 56 empty lines

File: templates/addCourse.phtml [hunk 3]
```diff
@@ -39,13 +39,25 @@ $trainersCheckbox = \Portal\ViewHelpers\TrainerCheckboxViewHelper::displayTraine
...
+
+            <input id="remote_input" name="remote_input" class="create-courses form-control" type="text" data-required="true" />
+            <div id="remote_input_error" class="alert formItem_alert"></div>
+
```
Comment (Joperes-ym) [inline templates/addCourse.phtml:59]: Clear 59 or 60 empty lines

**Pull request merged**